blackbox/docs/sql/reference/select.txt

```diff
@@ -242,6 +243,34 @@ resulting row of a group by clause.
    Grouping can only be applied on indexed fields. For more information, please
    refer to :ref:`sql_ddl_index_off`.
 
+.. _sql_reference_union:
+
+UNION ALL Clause
+................
+
+The UNION ALL operator computes the set union of the rows returned by the
```

Comment:
I would say it like this to be more understandable "The UNION ALL operator combines the result set of two or more SELECT statements."